### ISSUE TITLE
CRS correctness

### ISSUE THREAD

**Issue by evetion** [JuliaGeo/GeoJSON.jl#80]
#sdsl23

We should at least warn if you provide a spatial thing with a (defined?) crs other than wgs84.

**Comment by visr** [comment]
Perhaps error with a way to opt out if it is another defined CRS, and accept when there is no CRS or CRS WGS84.

> However, where all involved parties have a prior arrangement, alternative coordinate reference systems can be used without risk of data being misinterpreted."

**Comment by rafaqz** [comment]
Error with manual override sounds best to me. 

But do we attach the crs following the old api if users force it?

**Comment by visr** [comment]
I'd say no, at least not without explicitly opting in to writing the old standard, if we ever decide to support it. It's simpler for us to stick to the RFC standard.

**Comment by rafaqz** [comment]
Yeah easier for us for sure. I guess people can use GDAL if they want the old standard and to actually write the crs.
